bin/lerna.js

```diff
@@ -10,25 +10,26 @@ var cli = meow([
   "  $ lerna [command]",
   "",
   "Commands:",
-  "  bootstrap  Link together local packages and npm install remaining package dependencies",
-  "  publish    Publish updated packages to npm",
-  "  updated    Check which packages have changed since the last release",
-  "  import     Import a package with git history from an external repository",
-  "  clean      Remove the node_modules directory from all packages",
-  "  diff       Diff all packages or a single package since the last release",
-  "  init       Initialize a lerna repo",
-  "  run        Run npm script in each package",
-  "  exec       Run a command in each package",
-  "  ls         List all public packages",
+  "  bootstrap              Link together local packages and npm install remaining package dependencies",
+  "  publish                Publish updated packages to npm",
+  "  updated                Check which packages have changed since the last release",
+  "  import                 Import a package with git history from an external repository",
+  "  clean                  Remove the node_modules directory from all packages",
+  "  diff                   Diff all packages or a single package since the last release",
+  "  init                   Initialize a lerna repo",
+  "  run                    Run npm script in each package",
+  "  exec                   Run a command in each package",
+  "  ls                     List all public packages",
+  "  watch [glob] [script]  Watch for changes to package files and run a script when a change is detected",
```

Comment:
Oy, we really have to get away from this fiddly manual formatting. (urge to refactor into [yargs](https://github.com/yargs/yargs/) rising...)

None of the other commands list their positional arguments here, can we stay consistent with that until we move to a better argparser?